plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/delete/PositionDeleteWriter.java

```diff
@@ -88,26 +83,7 @@ public IcebergPositionDeletePageSink(
         this.writer = fileWriterFactory.createPositionDeleteWriter(fileSystem, Location.of(outputPath), session, fileFormat, storageProperties);
     }
 
-    @Override
-    public long getCompletedBytes()
-    {
-        return writer.getWrittenBytes();
-    }
-
-    @Override
-    public long getMemoryUsage()
```

Comment:
Are we tracking memory usage by the position deletes file writer anywhere ?

Reply:
No, they are created inline during `finish()`.